Modules/HMPID/src/clusters.json

```diff
@@ -0,0 +1,59 @@
+{
+   "qc":{
+      "config":{
+         "database":{
+            "implementation":"CCDB",
+            "host":"ccdb-test.cern.ch:8080",
+            "username":"not_applicable",
+            "password":"not_applicable",
+            "name":"not_applicable"
+         },
+         "Activity":{
+            "number":"42",
+            "type":"2"
+         },
+         "monitoring":{
+            "url":"infologger:///debug?qc"
+         },
+         "consul":{
+            "url":"http://consul-test.cern.ch:8500"
+         },
+         "conditionDB":{
+            "url":"ccdb-test.cern.ch:8080"
+         }
+      },
+      "tasks":{
+         "TaskClusters":{
+            "active":"true",
+            "className":"o2::quality_control_modules::hmpid::HmpidTaskPippo",
```

Comment:
I don't think that this class name is correct.

Reply:
Of course is wrong :)!